inference-engine/thirdparty/movidius/mvnc/tests/mvnc_tests_usb.cpp

```diff
@@ -28,7 +28,7 @@ TEST_F(MvncOpenUSBDevice, ShouldOpenDeviceAfterChangeConnectTimeoutFromZero) {
 
     ASSERT_NO_ERROR(ncSetDeviceConnectTimeout(0));
     ASSERT_ERROR(ncDeviceOpen(&deviceHandle, deviceDesc, m_ncDeviceOpenParams));
-    std::this_thread::sleep_for(3_sec);
+    std::this_thread::sleep_for(5_sec);
```

Comment:
Why sleep should be 5 seconds? Is it just empirical observation?

Reply:
By default, timeout is 15 seconds:
`static int g_deviceConnectTimeoutSec = 15;`
But in test we do not have access to this constant. Also, yes, locally 5 seconds is enough for Window. (for Linux was enough 3 seconds)

Reply:
15 seconds also is "empirical observation" :)

Reply:
Ok, looks good then